src/java.desktop/unix/native/common/awt/awt_GraphicsEnv.h

```diff
@@ -50,7 +50,7 @@
 #define MITSHM_PERM_COMMON (0666)
 #define MITSHM_PERM_OWNER  (0600)
 
-extern int XShmQueryExtension();
+extern int XShmQueryExtension(Display* dpy);
```

Comment:
I wonder if it's even needed here: the declaration should come from `X11/extensions/XShm.h` which is included.

Reply:
I agree, but I wonder why it was ever added ? Near as I can tell its been there since the SHM code was added in 1.4

Reply:
So I have removed the declaration.